web/themes/custom/pineapple/sass/components/paragraphs/_swiper.scss

```diff
@@ -0,0 +1,48 @@
+.swiper-container {
```

Comment:
Style currently targets any and all swiper containers that may be added to the site, consider replacing this with the paragraph name as you've done in your js file and also rename the file to match that change.

Reply:
done.